packages/main/src/RadioButtonGroup.ts

```diff
@@ -1,33 +1,40 @@
+import RadioButton from "./RadioButton.js";
```

Comment:
Is it working with "import type RadioButton", just want to make sure it's needed as now
the RadioButton imports the RadioButtonGroup and RadioButtonGroup imports the RadioButton and if the "import type" works it would be better